CONTRIBUTING.adoc

```diff
@@ -68,7 +68,7 @@ https://redhat-documentation.github.io/modular-docs/[Modular Documentation Initi
 
 .Procedure
 
-. Open a che-docs workspace running on Hosted Che: `++https++://che.openshift.io/factory?url=https://github.com/__<github-handle>__/che-docs/tree/__<branch-name>__/`
+. Open a che-docs workspace running on Hosted Che: `++https++://workspaces.openshift.com/#https://github.com/__<github-handle>__/che-docs/tree/__<branch-name>__/`
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on `f?url=` see https://github.com/eclipse/che-docs/pull/1884